cli/src/format.rs

```diff
@@ -54,7 +47,7 @@ pub enum Output {
 }
 
 impl Output {
-    pub fn new(path: Option<&Path>) -> Result<Self, FormatError> {
+    pub fn new(path: Option<&Path>) -> CliResult<Self> {
```

Comment:
Since `new` seems to be called in two different code paths below with definite value `None` and `Some(path)`, would that be clearer to have a `stdout()` and a `file(path)` instead (or `new_stdout()` and `new_from_path(path)` or whatnot) ?